metricbeat/module/elasticsearch/enrich/_meta/fields.yml

```diff
@@ -15,11 +15,62 @@
           type: long
           description: >
             Current number of outstanding remote requests.
-        - name: total
+        - name: count
           type: long
           description: >
             Number of outstanding remote requests executed since node startup.
-    - name: executed_searches.total
+    - name: executed_searches.count
       type: long
       description: >
         Number of search requests that enrich processors have executed since node startup.
+    - name: coordinator_stats
+      type: group
+      fields:
+        - name: queue.size
+          type: long
+          description: TODO
+        - name: remote_requests
+          type: group
+          fields:
+            - name: current
+              type: long
+              description: TODO
+            - name: count
+              type: long
+              description: TODO
+        - name: executed_searches.count
+          type: long
+          description: TODO
+    - name: executing_policy
+      type: group
+      fields:
+        - name: name
+          type: keyword
+          description: TODO
+        - name: task
+          type: group
+          fields:
+            - name: id
+              type: long
+              description: TODO
+            - name: type
+              type: keyword
+              description: TODO
+            - name: action
+              type: keyword
+              description: TODO
+            - name: time
+              type: group
+              fields:
+                - name: start.ms
+                  type: long
+                  description: TODO
+                - name: running.nano
+                  type: long
+                  description: TODO
+            - name: cancellable
+              type: boolean
+              description: TODO
+            - name: parent_task_id
+              type: keyword
+              description: TODO
```

Comment:
Given what @chrisronline said about Stack Monitoring not actually using this metricset, I would suggest NOT adding all these extra fields just yet, for two reasons:

1. It gives us flexibility to add them, as needed, in the future. It's always easy to add more fields than take away, as taking away represents a breaking change. Also, there's a cost to storing fields so only storing the ones we are actually using seems prudent.

2. You won't need to flesh out all the TODOs for the field descriptions. 🙂

Reply:
You gained my heart with point number 2 :smile: I've removed them